features/writing-mode.yml.dist

```diff
@@ -0,0 +1,52 @@
+# Generated from: writing-mode.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2017-03-27
+  baseline_high_date: 2019-09-27
+  support:
+    chrome: "48"
+    chrome_android: "48"
+    edge: "12"
+    firefox: "41"
+    firefox_android: "41"
+    safari: "10.1"
+    safari_ios: "10.3"
+compat_features:
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: high
+  # baseline_low_date: 2017-03-27
+  # baseline_high_date: 2019-09-27
+  # support:
+  #   chrome: "48"
+  #   chrome_android: "48"
+  #   edge: "12"
+  #   firefox: "41"
+  #   firefox_android: "41"
+  #   safari: "10.1"
+  #   safari_ios: "10.3"
+  - css.properties.writing-mode
+  - svg.global_attributes.writing-mode
+
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "48"
+  #   chrome_android: "48"
+  #   edge: "79"
+  #   firefox: "43"
+  #   firefox_android: "43"
+  #   safari: "10.1"
+  #   safari_ios: "10.3"
+  - css.properties.writing-mode.horizontal-tb
+  - css.properties.writing-mode.vertical-lr
+  - css.properties.writing-mode.vertical-rl
```

Comment:
Not a blocker but I've sent https://github.com/mdn/browser-compat-data/pull/24517 to fix this.